pmoves/tools/beats_to_voice.py

```
@@ -117,6 +120,70 @@ def _check_flute_health(base_url: str) -> bool:
+# NATS publish / subscribe
+# ---------------------------------------------------------------------------
+
+async def _nats_publish_cgp(cgp_packet: dict, nats_url: str = NATS_URL) -> bool:
+    """Publish CGP packet to tokenism.prosodic.bpm.v1. Returns True on success."""
+    try:
+        import nats as natspy  # optional dep — lazy import, same pattern as beats_to_cgp.py
+        nc = await natspy.connect(nats_url)
+        await nc.publish(NATS_SUBJECT, json.dumps(cgp_packet).encode("utf-8"))
+        await nc.drain()
+        return True
+    except Exception as e:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)